.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
```

Comment:
Hi @sullis,
This is a Gradle project, did you get to test it to make sure it works?

Reply:
Good catch. I just updated the file.   I changed "maven" to "grade"